docs/api/select.md

```diff
@@ -130,6 +130,14 @@ Customizing the interface dialog should be done by following the Customization s
 
 However, the Select Option does set a class for easier styling and allows for the ability to pass a class to the overlay option, see the [Select Options documentation](select-option.md) for usage examples of customizing options.
 
+## Typeahead Component
+
+Typeahead or autocomplete functionality does not exist in `ion-select` but can be built using existing Ionic components. We recommend using an `ion-modal` to make the best use of the available screen space.
+
+import TypeaheadExample from '@site/static/usage/select/typeahead/index.md';
+
+<TypeaheadExample />
```

Comment:
Had a couple nitpicks with the styling when a value is selected. On `ios` mode, the selected value is gray like the placeholder, instead of black like in a regular `ion-select`:

![ios mode](https://i.gyazo.com/c8f38e09c0684e77d679770424f6a3c3.png)

And on `md` mode, the font is unusually small:

![md mode](https://i.gyazo.com/0075a777a666cd265f757adc99e5c1c3.png)

Would it be possible to adjust these?